### PR TITLE
Update Acra tools with kms cli

### PR DESCRIPTION
**Pull request by ZhmakaAS**
Updated Acra tools with latest KMS CLI flags to support MasterKey loading.

## Checklist

- [ ] Change is covered by automated tests
- [ ] The [coding guidelines] are followed
- [ ] Public API has proper documentation in the [Acra documentation] site or has PR on [documentation repository] 
  with new changes
- [ ] CHANGELOG.md is updated (in case of notable or breaking changes)
- [ ] CHANGELOG_DEV.md is updated
- [ ] Benchmark results are attached (if applicable)
- [ ] [Example projects and code samples] are up-to-date (in case of API changes) 

[coding guidelines]: https://golang.org/doc/effective_go
[Example projects and code samples]: https://github.com/cossacklabs/acra-engineering-demo
[Acra documentation]: https://docs.cossacklabs.com/
[documentation repository]: https://github.com/cossacklabs/product-docs

### CODE REVIEW

**Comment by Lagovas** [comment]
can we check the operational status of AT somehow? verify that it can start and respond to any request? because AT and AS are the most essential binaries.

File: tests/test.py [hunk 1]
```diff
@@ -4695,6 +4713,10 @@ def testApiEncryptionDisableCacheWithoutKeysDir(self):
                 deserialize_and_decrypt_acrastruct(response, client_id_private_key, client_id)
 
 
+class TestTranslatorDisableCachedOnStartupWithAWSKMS(AWSKMSMasterKeyLoaderMixin, TestTranslatorDisableCachedOnStartup):
+    pass
```
Comment (Lagovas) [inline tests/test.py:4717]: add here a comment about the internal logic of master key loading too, please.

**Review by Lagovas — APPROVED** [review]
lgtm but add comment please to last test why it is empty
